esp/public/media/scripts/custom_form.js

```diff
@@ -865,6 +868,8 @@ var onSelectField=function($elem, field_data, ftype=null) {
 	//Select the current field and category in the field and category selectors
 	if(ftype in formElements['Generic']){
 		$j('#cat_selector').val('Generic');
+        // Don't allow changing a field into a page or section (instructions are ok, though)
+        disabled_fields=['page', 'section'];
```

Comment:
So instead of disabling the option in the drop down list for the field type, you can still click section or page, but then once you do that, both the field category and field dropdowns are entirely disabled. THis means you can't undo what you just did and select a different field category or field type without unselecting and reselecting the item in the preview. Is it possible to disable the section and page options within the drop down before you click it instead of disabling the whole dropdown afterwards? or if you cant do that, when someone selects a bad option, then leave the field category box enabled so they can at least switch back to an allowable category? this still wouldnt allow them to switch to instructions without switching the category away and back though, so disabling individual options within the dropdown would still be preferable.